.github/workflows/ci.yml

```
@@ -108,8 +106,14 @@ jobs:
     runs-on: macos-14
     steps:
     - uses: actions/checkout@v6
+    - name: Create and set the default keychain
+      run: |
+        security create-keychain -p "" temporary
+        security default-keychain -s temporary
+        security unlock-keychain -p "" temporary
+        security set-keychain-settings -lut 7200 temporary
     - name: Build-Test
-      run: set -o pipefail && env NSUnbufferedIO=YES xcodebuild -workspace Parse.xcworkspace -scheme ParseSwift -destination platform\=iOS\ Simulator,name\=iPhone\ 15\ Pro\ Max -derivedDataPath DerivedData test 2>&1 | xcbeautify --renderer github-actions
+      run: set -o pipefail && env NSUnbufferedIO=YES swift test -v #--enable-code-coverage 2>&1 | xcbeautify --renderer github-actions
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)